Geometry/HGCalCommonData/src/HGCalDDDConstants.cc

```diff
@@ -1338,16 +1338,16 @@ void HGCalDDDConstants::waferFromPosition(const double x,
       if (ktr != hgpar_->waferInfoMap_.end()) {
         place = HGCalCell::cellPlacementIndex(1, HGCalTypes::layerFrontBack(layertype), (ktr->second).orient);
         part = (ktr->second).part;
+#ifdef EDM_ML_DEBUG
         if (debug)
```

Comment:
Why do you still need this `if`?

Reply:
This is a time-critical code. So avoid the testing for standard processing - and once compiled, need to depend on the debug flag